<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanApplications</class>
 <widget class="QWidget" name="LoanApplications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Поиск:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="searchColumn">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {    
	background-color: #0E9252;    
	border: 1px solid #2980b9;    
	color: #ffffff;    
	padding: 5px;    
	border-radius: 3px;
}

QComboBox:hover {   
	background-color: #42A977;    
	border: 1px solid #1c6da5;
}

QComboBox:disabled {   
	background-color: #d3d3d3;    
	color: #555555;   
	border: 1px solid #a3a3a3;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchText">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Точное 
совпадение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}

QPushButton:hover {
	background-color: #2184cb;
	border: 1px solid #1c6da5;
}

QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/search.png</normaloff>:/assets/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSearch">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}

QPushButton:hover {
	background-color: #2184cb;
	border: 1px solid #1c6da5;
}

QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/clearSearch.png</normaloff>:/assets/clearSearch.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортировать столбец:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortingColumn">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {    
	background-color: #0E9252;    
	border: 1px solid #2980b9;    
	color: #ffffff;    
	padding: 5px;    
	border-radius: 3px;
}

QComboBox:hover {   
	background-color: #42A977;    
	border: 1px solid #1c6da5;
}

QComboBox:disabled {   
	background-color: #d3d3d3;    
	color: #555555;   
	border: 1px solid #a3a3a3;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeSorting">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {    
	background-color: #0E9252;    
	border: 1px solid #2980b9;    
	color: #ffffff;    
	padding: 5px;    
	border-radius: 3px;
}

QComboBox:hover {   
	background-color: #42A977;    
	border: 1px solid #1c6da5;
}

QComboBox:disabled {   
	background-color: #d3d3d3;    
	color: #555555;   
	border: 1px solid #a3a3a3;
}</string>
       </property>
       <item>
        <property name="text">
         <string>По возрастанию</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По убыванию</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sorting">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортировать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Текущая страница:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCurrentPage">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxPage">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>????</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}

QPushButton:hover {
	background-color: #2184cb;
	border: 1px solid #1c6da5;
}

QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
       </property>
       <property name="text">
        <string> &lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Перейти к странице №:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pageNumberToNavigate">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #3498db;
	border: 1px solid #2980b9;
	color: #ffffff;
	padding: 5px 10px;
	border-radius: 3px;
}

QPushButton:hover {
	background-color: #2184cb;
	border: 1px solid #1c6da5;
}

QPushButton:pressed {
	background-color: #1a548b;
	border: 1px solid #174172;
}

QPushButton:disabled {
	background-color: #d3d3d3;
	color: #555555;
	border: 1px solid #a3a3a3;
}</string>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="numberRows_10">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="numberRows_15">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="numberRows_20">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="automaticNumberRows">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Авто</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
